diff --git a/include/index/IndexFactory.h b/include/index/IndexFactory.h
index 918dd12..53c2447 100644
--- a/include/index/IndexFactory.h
+++ b/include/index/IndexFactory.h
@@ -54,6 +54,7 @@ private:
 	 */
 	enum IndexType {
 		INVALID_INDEX, /**< Unknown/Invalid power index */
+		NI,			   /**< new index */
 		ABZ,		   /**< absolute Banzhaf */
 		APG,		   /**< absolute Public Good*/
 		APIG,		   /**< absolute null-player-free Power Index G (based on the Dubey-Shapley identity)*/
diff --git a/include/index/NewIndex.h b/include/index/NewIndex.h
new file mode 100644
index 0000000..7db9ec1
--- /dev/null
+++ b/include/index/NewIndex.h
@@ -0,0 +1,26 @@
+#ifndef EPIC_INDEX_NEWINDEX_H_
+#define EPIC_INDEX_NEWINDEX_H_
+
+#include "RawPublicGood.h"
+
+namespace epic::index {
+
+/**
+ * Add here a description of the new power index for the Doxygen documentation generator.
+ */
+class NewIndex : public RawPublicGood {
+/*
+ * Override the RawPublicGood methods as required:
+ */
+public:
+	NewIndex(Game& g, ItfUpperBoundApproximation* approx, IntRepresentation int_representation = DEFAULT);
+
+	std::vector<bigFloat> calculate() override;
+	std::string getFullName() override;
+
+	// longUInt getMemoryRequirement() override; // only if your memory requirement differs from the RawPublicGood
+};
+
+} // namespace epic::index
+
+#endif //EPIC_INDEX_NEWINDEX_H_
diff --git a/readme.md b/readme.md
index 4f3e506..c29725a 100644
--- a/readme.md
+++ b/readme.md
@@ -57,6 +57,7 @@ First open a terminal and go to the `epic/` directory: `cd <path_to>/epic`. Afte
 
 | shortcut | name |
 | -------- | ---- |
+| `NI` | New Index |
 | `ABZ` | Absolute Banzhaf |
 | `BZ` | Banzhaf |
 | `CC` | Coleman Collective |
diff --git a/src/index/IndexFactory.cpp b/src/index/IndexFactory.cpp
index f5e5dfe..e6ceb91 100644
--- a/src/index/IndexFactory.cpp
+++ b/src/index/IndexFactory.cpp
@@ -1,6 +1,7 @@
 #include "index/IndexFactory.h"
 
 #include "FastUpperBoundApproximation.h"
+#include "index/NewIndex.h"
 #include "index/AbsoluteBanzhaf.h"
 #include "index/AbsolutePublicGood.h"
 #include "index/AbsolutePowerIndexG.h"
@@ -42,6 +43,7 @@
 
 const std::map<epic::index::IndexFactory::IndexType, std::pair<std::string, std::string>> epic::index::IndexFactory::cIndexNames = {
 	{INVALID_INDEX, {"INVALID", "invalid index"}},
+	{NI, {"NI", "new index"}},
 	{ABZ, {"ABZ", "absolute Banzhaf"}},
 	{APG, {"APG", "absolute Public Good"}},
 	{APIG, {"APIG", "absolute null-player-free Power Index G (based on the Dubey-Shapley identity)"}},	
@@ -92,6 +94,7 @@ epic::index::ItfPowerIndex* epic::index::IndexFactory::new_powerIndex(const std:
 
 	switch (getIndexType(short_name)) {
 		case IndexType::INVALID_INDEX: index = nullptr; break;
+		case IndexType::NI: index = new NewIndex(g, approx, int_representation); break;
 		case IndexType::ABZ: index = new AbsoluteBanzhaf(g, approx, int_representation); break;
 		case IndexType::BZ: index = new Banzhaf(g, approx, int_representation); break;
 		case IndexType::APG: index = new AbsolutePublicGood(g, approx, int_representation); break;
diff --git a/src/index/NewIndex.cpp b/src/index/NewIndex.cpp
new file mode 100644
index 0000000..e63b7c3
--- /dev/null
+++ b/src/index/NewIndex.cpp
@@ -0,0 +1,22 @@
+#include "index/NewIndex.h"
+
+epic::index::NewIndex::NewIndex(Game& g, ItfUpperBoundApproximation* approx, IntRepresentation int_representation) : RawPublicGood(g, approx, int_representation) {
+	/*
+	 * If needed add additional initializations below
+	 */
+}
+
+std::vector<epic::bigFloat> epic::index::NewIndex::calculate() {
+	/*
+	 * Use mGame, mCalculator and the methods offered by RawPublicGood to calculate the power index and return the results.
+	 */
+	std::vector<bigFloat> solution = RawPublicGood::calculate();
+	return solution;
+}
+
+std::string epic::index::NewIndex::getFullName() {
+	/*
+	 * Return the full name of the implemented power index.
+	 */
+	return "NewIndex";
+}
